packages/venia-concept/src/ServiceWorker/Utilities/__tests__/htmlHandler.spec.js

```diff
@@ -115,26 +127,128 @@ test('fetchDidSucceed should call sendMessageToWindow if the response is differe
 
     expect(sendMessageToWindow).toHaveBeenCalledTimes(1);
     expect(sendMessageToWindow).toHaveBeenCalledWith(HTML_UPDATE_AVAILABLE);
+});
 
+test('fetchDidSucceed should not call sendMessageToWindow is the response has same title as that in the cache', async () => {
     /**
-     * Restoring prototype function definitions
+     * Mocking Request to return same value when .text is called.
      */
-    global.Response.prototype.text = originalText;
-    global.Response.prototype.clone = originalClone;
+    global.Response.prototype.text = function() {
+        return Promise.resolve('<html><title>Same Title</title></html>');
+    };
+    global.Response.prototype.clone = function() {
+        return new Response();
+    };
+
+    /**
+     * Mocking cache to have a response for 'https://develop.pwa-venia.com/'
+     */
+    matchFn.mockReturnValue(Promise.resolve(new Response()));
+
+    const url = 'https://develop.pwa-venia.com/';
+    const request = new Request(url);
+    const response = new Response();
+
+    await cacheHTMLPlugin.fetchDidSucceed({ request, response });
+
+    expect(sendMessageToWindow).not.toHaveBeenCalled();
 });
 
-test('fetchDidSucceed should not call sendMessageToWindow if the response if same as that in the cache', async () => {
+test("fetchDidSucceed should call sendMessageToWindow if 1 or more script tag's src has changed", async () => {
```

Comment:
```suggestion
test("fetchDidSucceed should call sendMessageToWindow if one or more script tags' src has changed", async () => {
```

Super minor but the convention is to spell out numbers less than 10.

I'm really not trying to be a pain I promise 😆

Reply:
I believe you Andy 😆